### DISCUSSION TITLE
Different with nuitka

### DISCUSSION THREAD

**Discussion by cyw3** [beeware/briefcase#1040] [Q&A]
### What is the problem or limitation you are having?

Hi,
I want to know [briefcase](https://briefcase.readthedocs.io/)'s difference with [Nuitka](https://github.com/Nuitka/Nuitka).

Thanks!

### Describe the solution you'd like

None

### Describe alternatives you've considered

None

### Additional context

None

**Comment by freakboy3742** [comment]
(Converted to a discussion since this wasn't an enhancement request).

The fundamental difference: nuitka produces a binary; Briefcase produces an app. This can be a subtle distinction, but it's an important one.

A binary is a single file; it's usually executed at a command line, and generally has no manifestation on a graphical desktop. `gcc` or `clang` are examples of binaries.

An App is composed of multiple files, can be installed, can ask for operating system permissions, and more. It will manifest as an icon on task managers. An IDE like Visual Studio or Xcode is an App.

As an additional benefit, Briefcase is able to support platforms like iOS, Android, and the web - platforms that Nuitka doesn't support.
